consensus/consensus.md

```diff
@@ -177,6 +180,20 @@ There is no need to consider Key Blocks' headers as Key Blocks do not hold heavy
 
 #### Micro Blocks
 
+```
+PROTOCOL_VERSION: 21
+GENESIS_VERSION: 21
+```
+
+- the timestamp of a micro block MUST be smaller than `now() + 10m`
```

Comment:
I see from code "smaller" means "strictly smaller". Though the concept of now() makes this detail not strictly needed in doc. No change needed.